seeds/quickbooks__financial_statement_ordinal.csv

```diff
@@ -0,0 +1,6 @@
+report,account_class,account_type,account_sub_type,account_number,ordinal
```

Comment:
Can you also add a seed.yml to document this seed file and the columns within it. [Here](https://docs.getdbt.com/reference/seed-properties) is how you can document it. Feel free to either create a new yml or integrate within an existing one yml file.